src/v2/Apps/Consign/consignRoutes.tsx

```diff
@@ -121,26 +112,41 @@ const prepareSubmissionFlowStepVariables = data => {
 
 export const consignRoutes: AppRouteConfig[] = [
   {
-    path: "/sell2",
+    path: "/sell",
     getComponent: () => NewMarketingLandingApp,
     onClientSideRender: () => {
       NewMarketingLandingApp.preload()
     },
   },
   {
     path: "/consign",
-    getComponent: () => MarketingLandingApp,
-    onClientSideRender: () => {
-      MarketingLandingApp.preload()
-    },
+    children: [
+      {
+        path: ":splat*",
+        render: ({ match }) => {
+          throw new RedirectException(
+            `${match.location.pathname.replace("/consign", "/sell")}${
+              match.location.search
+            }`,
+            301
+          )
+        },
+      },
+      {
+        path: "/",
+        render: ({ match }) => {
+          throw new RedirectException("/sell", 301)
+        },
+      },
+    ],
```

Comment:
@damassi I used 2 subroutes `:splat` and `/` for the redirecting all requests starter with `/consign` to the new `/sell` route. I was thinking about using `/consign(.*)?` path for that but looks like it's not working because versions of `path-to-regexp` are different for the app and Express. In the app, we use `6.1.0`, Express uses `0.1.7`. I tested the route via http://forbeslindesay.github.io/express-route-tester/ and path `/consign(.*)?` match only `/consign` and not match paths like `/consign/foo/bar` for the `0.1.7` version, but works fine for the subsequent versions started from `1.7.0.`.
Looks like it is required to update the Express version to use `(.*)` expression.
Please let me know if you have a better solution for that.

Reply:
Also, the problem can't be caught by unit tests because in the app we use an actual version of `path-to-regexp`

Reply:
Unfortunately, I was too optimistic, the latest Express version `4.18.1` still uses `path-to-regexp@0.1.7`. The `5.x` version is still in beta and doesn't use path-to-regexp at all. I'll check one more idea but not sure that will help with the problem.